test/helper.rb

```diff
@@ -21,7 +21,7 @@ def jruby?
 require 'minitest/reporters'
 require 'minitest/profile'
 require 'rspec/mocks'
-require 'jekyll'
+require File.expand_path("../lib/jekyll.rb", __dir__)
```

Comment:
`require_relative` it's been so long I can't remember but I think it's been there since 1.9 so you should be able to use it.